languages/julia/exercises/concept/boolean-logic/runtests.jl

```diff
@@ -0,0 +1,62 @@
+using Test
+
+include("game-logic.jl")
+
+@testset "fast attack" begin
+    @test !can_do_fast_attack(true)
+    @test  can_do_fast_attack(false)
+end
+
+@testset "spying" begin
+    character_state_combinations = [
```

Comment:
This raises the question whether the test suites should not require concepts the student doesn't know yet. For the web-based test runner, they're not visible so it makes no difference, but for someone solving them locally, it does. @ErikSchierboom or @iHiD do you have any guidance on this?

Reply:
Even using the CLI, the person should be able to solve it based on the README, so I think this is ok.

There's a **whole** question about skipping/unskipping tests for v3 that we need to work out at some point, but we can kick that down the road for now.

Reply:
> This raises the question whether the test suites should not require concepts the student doesn't know yet. For the web-based test runner, they're not visible so it makes no difference, but for someone solving them locally, it does.

In general, I'd avoid any syntax/concepts that the student does not know of in the test suite. In fact, there is even something on that in [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#tests-file). The existing (v2) Ruby track had one exercise where the output was defined using syntax I did not know about (some string formatting stuff) and that was incredibly frustrating as I couldn't easily see what the expected output was. 

So all in all, try to avoid using syntax/concepts the student doesn't know about, unless you have a very good reason not to (e.g. that a test would otherwise require tons of code to write).

Reply:
OK, I'll change them to be copy pasted tests then.

@SleeplessByte you may want to adjust the js test suite as well ^

Reply:
Personally I don't think I'd want to change the tests. They're supposed to solve the exercise using the instructions (which is perfectly doable), and not by reading the tests. In other words: sure it be great if we can keep the tests simple, but I also think the tests (in this case) don't need to be completely accessible 🤷  (I also don't want to give away the answers or make the DX super shitty).